recipes/frogress/meta.yaml

```
@@ -0,0 +1,41 @@
+{% set name = "frogress" %}
+{% set version = "0.9.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: da5b8206bce00ae245002975f50b73362e5d2e81151b48c66cfd4b15ba28952c
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  host:
+    - python >=3.7
+    - pip
+  run:
+    - python >=3.7
```

The `fcntl` module is not available on Windows but this package is installable there and would be broken.
You can use the virtual `__unix` package here to prevent this `noarch` from getting installed n Windows.

```suggestion
    - python >=3.7
    - __unix
```